modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -5720,7 +5720,7 @@
               "application/json": {
                 "example": {
                   "data": {
-                    "id": "9658fd0d-827e-4dbf-bf6f-07c9a0d82957",
```

Comment:
are all of these id, date, and pointer changes needed?

Reply:
Those are generated automatically by the rswag documentation process.